administration/src/application/components/ApplyController.tsx

```diff
@@ -9,18 +9,42 @@ import useVersionedLocallyStoredState from '../useVersionedLocallyStoredState'
 import DiscardAllInputsButton from './DiscardAllInputsButton'
 import { useGarbageCollectArrayBuffers, useInitializeGlobalArrayBuffersManager } from '../globalArrayBuffersManager'
 import ApplicationForm from './forms/ApplicationForm'
-import { useCallback, useMemo } from 'react'
+import { useCallback, useMemo, useState } from 'react'
 import { SnackbarProvider, useSnackbar } from 'notistack'
+import styled from 'styled-components'
 import ApplicationErrorBoundary from '../ApplicationErrorBoundary'
+import { useAppToaster } from '../../components/AppToaster'
 
 // This env variable is determined by '../../../application_commit.sh'. It holds the hash of the last commit to the
 // application form.
 const lastCommitForApplicationForm = process.env.REACT_APP_APPLICATION_COMMIT as string
 
 export const applicationStorageKey = 'applicationState'
 
+const SuccessContent = styled.div`
+  white-space: pre-line;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 1rem;
+`
+
 const ApplyController = () => {
-  const [addBlueEakApplication, { loading }] = useAddBlueEakApplicationMutation()
+  const [formSubmitted, setFormSubmitted] = useState<boolean>(false)
+  const appToaster = useAppToaster()
+  const [addBlueEakApplication, { loading }] = useAddBlueEakApplicationMutation({
+    onError: error => {
+      console.error(error)
+      appToaster?.show({ intent: 'danger', message: 'Beim Absenden des Antrags is ein Fehler aufgetreten.' })
+    },
+    onCompleted: result => {
+      if (result) {
+        setState(() => ApplicationForm.initialState)
+        setFormSubmitted(true)
+      } else {
+        appToaster?.show({ intent: 'danger', message: 'Beim Absenden des Antrags is ein Fehler aufgetreten.' })
```

Comment:
Sorry I have another nit:
Could you please use the `enqueueSnackbar` that we get from `notistack`'s `useSnackbar` (which is already called in line 58) instead of `appToaster?.show`.
The reason is that `useAppToaster` uses the blueprint framework, whereas here for the application we use the @mui's alternative (and shouldn't use any blueprint stuff)

Reply:
adjusted